lib/src/sdk.ts

```diff
@@ -25,9 +25,12 @@ export type SwapInfo = {
   toAccountId: string;
   fromAmount: BigNumber;
   feeStrategy: FeeStrategy;
+  customFeeConfig?: {
+    [key: string]: BigNumber;
```

Comment:
Is it possible to narrow the manage key or is it too vague right now?

Reply:
There is not an custom config type defined by blockchain. Do you want create it there and export it here?

Reply:
Ok :)